addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,102 @@
+oc create -f xpaas-templates/eap70-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-sso-s2i.json -n openshift
+
+oc create -f xpaas-templates/jws30-tomcat7-basic-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-https-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-basic-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-https-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/karaf2-camel-amq-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-log-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-rest-sql-template.json -n openshift
+oc create -f xpaas-templates/karaf2-cxf-rest-template.json -n openshift
+
+oc create -f xpaas-templates/openjdk18-web-basic-s2i.json -n openshift
+
+oc create -f xpaas-templates/processserver63-amq-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-basic-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/spring-boot-camel-amq-template.json -n openshift
```

Comment:
@praveenkumar @hferentschik I would use "oc apply" in any case, as any user can contribute addons and it could conflict between what is provided OOTB and what he provides. I would try to avoid conflicts in what is provided OOTB as much as possible. User contributed addons are user's responsibility.

With regards to keeping or removing fis addon, the question boils down to the granularity you want to provide to the end user. Given that importing the imagestreams and templates seems not to be time consuming, I would have one xpaas addon that install everything, reducing complexity and maintenance efforts.

Then every user could just make a user addon more slim if they want/need.